Scene.hpp

```diff
@@ -1,7 +1,6 @@
 #ifndef SCENE_H_
 #define SCENE_H_
 
-#include "GameObject.hpp"
```

Comment:
forward declaration kan ook al in deze class, dan hoeft het niet in de private headers.